distribution/archives/build.gradle

```diff
@@ -295,13 +295,19 @@ subprojects {
   }
 }
 
+subprojects {
+  group = "org.elasticsearch.distribution.${name.startsWith("oss-") ? "oss" : "default"}"
```

Comment:
Why do we need this? These are no longer published (except for integ-test-zip), so they should not need a group.

Reply:
testclusters uses the group of the artifact when extracting to make sure the extracted distributions don't overlap. I'm questioning the approach but decided not to change it yet.  When using the download service with the ivy repo, the group is never used when resolving it, but does end up configured on the artifact and we pick it up when extracting. When we use the projects for freshly built artifacts, the group gets picked up from the project and has to match. The alternative is to add a different configuration for each distribution type, so we will only ever have one artifact in each configuration and then we could use the configuration name when extracting rather than the group to avoid this confusion.